temporal-opentracing/src/main/java/io/temporal/opentracing/internal/SpanFactory.java

```diff
@@ -63,15 +63,17 @@ public SpanFactory(OpenTracingOptions options) {
       String workflowId,
       String parentWorkflowId,
       String parentRunId) {
-    Map<String, String> tags =
-        ImmutableMap.of(
-            StandardTagNames.WORKFLOW_ID, workflowId,
-            StandardTagNames.PARENT_WORKFLOW_ID, parentWorkflowId,
-            StandardTagNames.PARENT_RUN_ID, parentRunId);
-    String operationName =
-        options.getSpanOperationNamePrefix(SpanOperationType.START_CHILD_WORKFLOW)
-            + PREFIX_DELIMITER
-            + childWorkflowType;
+    StartSpanContext context =
+        new StartSpanContext(
+            options,
+            SpanOperationType.START_CHILD_WORKFLOW,
+            childWorkflowType,
+            workflowId,
+            null,
+            parentWorkflowId,
+            parentRunId);
+    Map<String, String> tags = options.getOperationNameAndTagsProvider().getSpanTags(context);
```

Comment:
Please lookup options.getOperationNameAndTagsProvider() into a local var  once and use it, code will be cleaner. The same s true for other usages like this in this PR